Readme.md

```diff
@@ -57,6 +57,12 @@ Custom config and tasks folder:
 mrm license --dir ~/unicorn
 ```
 
+Run a task from a preset (globally installed `mrm-preset-unicorn` npm package):
```

Comment:
Wondering what a preset is when reading this 😉

Reply:
Don't know if it's too late to throw in that thought, but might also be nice to be able to run `mrm preset-unicorn` instead of `mrm --preset unicorn`

Reply:
> Wondering what a preset is when reading this 

I can link a section about presets here.

> mrm preset-unicorn instead of mrm --preset unicorn

Help me understand how is better? For me it’s unclear what it does, unexplicit, hard to implement (how should we know that `preset-unicorn` is a preset? What about scoped packages?), and not even shorter ;-|

Reply:
> I can link a section about presets here.
👍
> Help me understand how is better?
Just felt that `--preset unicorn` looks like an option, not like another kind of argument

Reply:
What do you mean by “an option” and “another kind of argument”. There’s only one argument which is not an option (`--NAME`) — a task name, and it can be repeated.

Reply:
My point is that the preset feels more like an argument than an option. We have things that can be run and configuration for them. In my head (might be a personal issue) *things that can run* are arguments and *configuration* are passed as CLI options or taken from some other file.

Reply:
But we can’t run a preset ;-) We can only run a task or alias which may be inside a preset. Treat `--preset` and `--dir` like modifiers. 

And this distinction between arguments and option is very subtle, I prefer to make things explicit then magical and implicit, especially when magic is hard to implement and (which is more important) hard to *explain*.

Reply:
I’ve updated the docs.